### PR TITLE
Rename `onlyUpdater` -> `onlyUpdaterRole`

### PR DESCRIPTION
**Pull request by MerlinEgalite**
Fixes https://github.com/cantinasec/review-morpho-blue-1/issues/114

### CODE REVIEW

File: src/UniversalRewardsDistributor.sol [hunk 1]
```diff
@@ -52,7 +52,7 @@ contract UniversalRewardsDistributor is IUniversalRewardsDistributor {
     }
 
     /// @notice Reverts if the caller is not the owner nor an updater.
-    modifier onlyUpdater() {
+    modifier onlyUpdaterRole() {
         require(isUpdater[msg.sender] || msg.sender == owner, ErrorsLib.CALLER_NOT_OWNER_OR_UPDATER);
```
Comment (Rubilmax) [inline src/UniversalRewardsDistributor.sol:56]:
```suggestion
        require(isUpdater[msg.sender] || msg.sender == owner, ErrorsLib.CALLER_NOT_OWNER_NOR_UPDATER);
```

Comment (MerlinEgalite) [reply src/UniversalRewardsDistributor.sol:56]: or `NOT_CURATOR_ROLE` like in MM?

Comment (Rubilmax) [reply src/UniversalRewardsDistributor.sol:56]: > or `NOT_CURATOR_ROLE` like in MM?

this = best

**Review by Rubilmax — APPROVED** [review]
I think error messages could be shortened but I'm not blocking ^^

**Pull request merged**